src/Json/Signer.cpp

```diff
@@ -0,0 +1,55 @@
+//
+// Created by Leone Parise on 2019-05-24.
+//
+
+#include "Signer.h"
+#include "Base64.h"
+#include "../Cosmos/Signer.h"
+#include <google/protobuf/util/json_util.h>
+
+using namespace TW;
+using namespace TW::Json;
```

Comment:
can we find a better name? `Json` is too general, unlike `Bitcoin` or other chains

Reply:
I know but it's generic, I can sign any blockchain. Do you know a better name? I'm open to suggestions.

Reply:
1) `Signer`, 
2) `TransactionSigner`
3) `ChainSigner`

Reply:
Any blockchains? AnySigner :)

Reply:
I liked AnySigner. It can sign any blockchain.

Reply:
I would use just `Signer` for simplicity

Reply:
Renamed on last commit.